compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUSplitReduction.cpp

```diff
@@ -4,6 +4,7 @@
 // See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+#include "iree/compiler/Codegen/LLVMCPU/KernelDispatch.h"
```

Comment:
It looks like we may want to move whatever is in this file that we use here to a common utility file?

Reply:
I think I'll rebase on your PR and use TilingConfig.

Reply:
Has this been rebased? SHould this be TileSizeSelection.h then?

Reply:
I don't know why it's not pushed. Updated.. sorry about that.